### DISCUSSION TITLE
`BarChart` and `Tooltip` back!

### DISCUSSION THREAD

**Discussion by fenix-hub** [fenix-hub/godot-engine.easy-charts#61] [General]
`BarChart` are now back, and a better `Tooltip` too!

https://user-images.githubusercontent.com/45271396/213925733-120c1339-6850-48b3-baf5-f269cec9eecc.mp4

The `Tooltip` will now show:
- `x value` of the point/column you are hovering on
- `color` associated to the point function
- `name` associated to the point function
- `y value` associated to the point function

**Full Changelog**: https://github.com/fenix-hub/godot-engine.easy-charts/compare/v22.01.17...v23.01.22

<hr /><em>This discussion was created from the release <a href='https://github.com/fenix-hub/godot-engine.easy-charts/releases/tag/v23.01.22'>`BarChart` and `Tooltip` back!</a>.</em>

**Comment by rs-v** [comment]
hello here!i 'm wandering impl like "day 1 day 2 day 3" labels  in the latest version, i test the example,but it doesn't work
![图片](https://github.com/user-attachments/assets/89c17b2d-06d2-4777-8291-9eb8f460a57c)

**Reply by fenix-hub** [reply]
Hi @rs-v ,
please, follow this PR #121
